examples/oit_reconstruction_new.py

```diff
@@ -0,0 +1,981 @@
+# Copyright 2014-2016 The ODL development group
```

Comment:
This file is way to long for an example, examples need to be at most 100 lines or so.

If we want to have examples this complicated, they should be moved to some external place (some `odl-examples` repo, for example).